extensions/amp-story-shopping/validator-amp-story-shopping.protoascii

```diff
@@ -19,24 +19,52 @@ html_format: AMP
   amp_layout: {
     supported_layouts: CONTAINER
   }
+  attrs: {
+    name: "data-product-id"
+    mandatory: true
+  }
 }
-tags: {  # <amp-story-shopping-config>
+tags: {  # <amp-story-shopping-attachment>
   html_format: AMP
-  tag_name: "AMP-STORY-SHOPPING-CONFIG"
+  tag_name: "AMP-STORY-SHOPPING-ATTACHMENT"
   requires_extension: "amp-story-shopping"
   attr_lists: "extended-amp-global"
   spec_url: "https://amp.dev/documentation/components/amp-story-shopping/"
+  attrs: {
+    name: "src"
+    value_url: {
+      protocol: "https"
+      allow_relative: true
```

Comment:
I don't we've decided quite yet if we will support relative urls will be for the launch.
I'm pretty sure we can get relative paths working. I'll hold off on removing this for now.

Reply:
We are going to have relative Url checks as we have built in utility functions to handle those.
They will be added with the validation PR here:
https://github.com/ampproject/amphtml/pull/37474

Reply:
Are you going to resolve the URL programmatically based on the origin URL and not the Cache URL? Otherwise fetching the configuration on cached pages will point to the cdn.ampproject.org URL vs the canonical one.